webrtc/README.md

```diff
@@ -11,170 +11,28 @@ Interest Group: [@marten-seemann]
 [@marten-seemann]: https://github.com/marten-seemann
 [@mxinden]: https://github.com/mxinden/
 
-<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
-**Table of Contents**
-
-- [WebRTC](#webrtc)
-  - [Motivation](#motivation)
-  - [Addressing](#addressing)
-  - [Connection Establishment](#connection-establishment)
-    - [Browser to public Server](#browser-to-public-server)
-  - [Multiplexing](#multiplexing)
-    - [Ordering](#ordering)
-    - [Head-of-line blocking](#head-of-line-blocking)
-    - [`RTCDataChannel` negotiation](#rtcdatachannel-negotiation)
-    - [`RTCDataChannel` label](#rtcdatachannel-label)
-  - [Connection Security](#connection-security)
-  - [Previous, ongoing and related work](#previous-ongoing-and-related-work)
-  - [Test vectors](#test-vectors)
-    - [Noise prologue](#noise-prologue)
-      - [Both client and server use SHA-256](#both-client-and-server-use-sha-256)
-- [FAQ](#faq)
+WebRTC flavors in libp2p:
 
-<!-- markdown-toc end -->
+1. [WebRTC browser-to-server](./browser-to-server.md)
 
-## Motivation
+   libp2p transport protocol **without the need for trusted TLS certificates.**
+   Enable browsers to connect to public server nodes without those server nodes
+   providing a TLS certificate within the browser's trustchain. Note that we can
+   not do this today with our Websocket transport as the browser requires the
+   remote to have a trusted TLS certificate. Nor can we establish a plain TCP or
+   QUIC connection from within a browser. We can establish a WebTransport
+   connection from the browser (see [WebTransport
+   specification](../webtransport)).
 
-1. **No need for trusted TLS certificates.** Enable browsers to connect to
-   public server nodes without those server nodes providing a TLS certificate
-   within the browser's trustchain. Note that we can not do this today with our
-   Websocket transport as the browser requires the remote to have a trusted TLS
-   certificate. Nor can we establish a plain TCP or QUIC connection from within
-   a browser. We can establish a WebTransport connection from the browser (see
-   [WebTransport specification](../webtransport)).
+2. [WebRTC W3C](./w3c.md)
 
-## Addressing
+  libp2p transport protocol using the W3C defined WebRTC connection
+  establishment flow, among other things enabling **hole punching in the
+  browser**.
 
-WebRTC multiaddresses are composed of an IP and UDP address component, followed
-by `/webrtc` and a multihash of the certificate that the node uses.
+## Shared
 
-Examples:
```

Comment:
The below, namely the browser-to-server specification, moved to `./browser-to-server.md`. The main `README.md` links to the two specs:

- webrtc browser-to-server introduced in https://github.com/libp2p/specs/pull/412 now in `./browser-to-server.md`
- webrtc w3c introduced in this pull request now in `./w3c.md`

Done with 4cca1f1.